features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -146,6 +146,11 @@
   "oauth.grant_type.device_code.enable": false,
   "oauth.grant_type.device_code.grant_handler": "org.wso2.carbon.identity.oauth2.device.grant.DeviceFlowGrant",
   "oauth.grant_type.device_code.grant_validator": "org.wso2.carbon.identity.oauth2.device.grant.DeviceFlowGrantValidator",
+  "oauth.grant_type.device_code.key_length": "6",
+  "oauth.grant_type.device_code.expiry_time": "600000",
+  "oauth.grant_type.device_code.polling_interval": "5000",
+  "oauth.grant_type.device_code.key_set": "BCDFGHJKLMNPQRSTVWXYZbcdfghjklmnpqrstvwxyz23456789",
```

Comment:
Any reason for not having `Aa1` here?

Reply:
In the specification it was recommended to remove vowels to avoid avoid randomly creating words. 1 and I also can be harder to detect. I think it is better to remove l as well. More details https://datatracker.ietf.org/doc/html/rfc8628#section-6.1

Reply:
In the specification it was recommended to remove vowels to avoid avoid randomly creating words. 1 and I also can be harder to detect. More details https://datatracker.ietf.org/doc/html/rfc8628#section-6.1